### PR TITLE
[2.x] OpenApi Generator examples

### PR DESCRIPTION
**Pull request by aserkes**
[2.x] backport for OpenApi Generator examples

### CODE REVIEW

File: examples/openapi-tools/quickstart-mp/README.md [hunk 1]
```diff
@@ -0,0 +1,296 @@
...
+
+## Build, prepare and run the Helidon MP server
+
+To generate Helidon MP server at first we create `mp-server` folder and then inside it we run the following command where `path-to-generator` is the directory where you downloaded the generator CLI JAR file and `path-to-openapi-doc` is the folder where `quickstart.yaml` is located:
+```bash
+java -jar path-to-generator/openapi-generator-cli.jar \
+          generate \
+          -g java-helidon-server \  
+          --library mp \
+          -p helidonVersion=2.5.2 \
```
Comment (tjquinno) [inline examples/openapi-tools/quickstart-mp/README.md:17]: Please change to 2.5.6 (the now-current release).

File: examples/openapi-tools/quickstart-mp/README.md [hunk 2]
```diff
@@ -0,0 +1,296 @@
...
+
+The second part of this example is generating MicroProfile Rest Client that will communicate with the server that we have just created.
+
+To generate Helidon MP client at first we create `mp-client` folder and then inside it we run the following command where `path-to-generator` is the directory where you downloaded the generator CLI JAR file and `path-to-openapi-doc` is the folder where `quickstart.yaml` is located:
+```bash
+java -jar path-to-generator/openapi-generator-cli.jar \
+          generate \
+          -g java-helidon-client \  
+          --library mp \
+          -p helidonVersion=2.5.2 \
```
Comment (tjquinno) [inline examples/openapi-tools/quickstart-mp/README.md:89]: 2.5.6

File: examples/openapi-tools/quickstart-mp/mp-client/pom.xml [hunk 3]
```diff
@@ -0,0 +1,135 @@
...
+-->
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <groupId>org.openapitools</groupId>
+    <parent>
+        <groupId>io.helidon.applications</groupId>
+        <artifactId>helidon-mp</artifactId>
+        <version>2.5.2</version>
```
Comment (tjquinno) [inline examples/openapi-tools/quickstart-mp/mp-client/pom.xml:27]: 2.5.7-SNAPSHOT

By changing the versions in the ``pom files to this, they will automatically be updated with each new release. Then we would just need to update the release in the readme files manually. I am seeing if there is a good way to update those automatically as well.. But not necessarily as part of this PR.

File: examples/openapi-tools/quickstart-mp/mp-server/pom.xml [hunk 4]
```diff
@@ -0,0 +1,134 @@
...
+
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <groupId>org.openapitools</groupId>
+    <parent>
+        <groupId>io.helidon.applications</groupId>
+        <artifactId>helidon-mp</artifactId>
+        <version>2.5.2</version>
```
Comment (tjquinno) [inline examples/openapi-tools/quickstart-mp/mp-server/pom.xml:29]: 2.5.7-SNAPSHOT

File: examples/openapi-tools/quickstart-se/se-client/pom.xml [hunk 5]
```diff
@@ -0,0 +1,131 @@
...
+-->
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <groupId>org.openapitools</groupId>
+    <parent>
+        <groupId>io.helidon.applications</groupId>
+        <artifactId>helidon-se</artifactId>
+        <version>2.5.2</version>
```
Comment (tjquinno) [inline examples/openapi-tools/quickstart-se/se-client/pom.xml:27]: 2.5.7-SNAPSHOT

File: examples/openapi-tools/quickstart-se/se-server/pom.xml [hunk 6]
```diff
@@ -0,0 +1,152 @@
...
+
+-->
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <groupId>io.helidon.applications</groupId>
+        <artifactId>helidon-se</artifactId>
+        <version>2.5.2</version>
```
Comment (tjquinno) [inline examples/openapi-tools/quickstart-se/se-server/pom.xml:26]: 2.5.7-SNAPSHOT

**Review by tjquinno — CHANGES_REQUESTED** [review]
We should change the Helidon version as noted (because I'm so late reviewing this).

**Comment by Captain1653** [comment]
I can be wrong, but there are many wrong copyrights in this PR. 
Screenshot 2023-02-24 at 22 24 38

**Comment by romain-grecourt** [comment]
@Captain1653 This is the PR was created in 2022 and merged in 2023.

**Pull request merged**